resources/rust/main.rs

```diff
@@ -1,9 +1,14 @@
 //! A blinky example for Tessel
 
+// This enables the system allocator in your script.
+// This is required to run Rust binaries on Tessel.
+#![feature(alloc_system)]
+extern crate alloc_system;
```

Comment:
In the [process of fixing on the build server](https://github.com/tessel/rust-compilation-server/pull/14).